src/api/java/de/teamlapen/vampirism/api/entity/player/actions/DefaultAction.java

```diff
@@ -10,16 +10,8 @@
  * Default implementation for an action
  */
 public abstract class DefaultAction<T extends IFactionPlayer> extends ForgeRegistryEntry<IAction> implements IAction {
-    private final IPlayableFaction<T> faction;
     private String translationKey;
 
-    /**
-     * @param faction
-     */
-    public DefaultAction(IPlayableFaction<T> faction) {
```

Comment:
This is still breaking.
Keep constructor and `getFaction` method.
Add additional empty constructor, that initializes with null and override `getFaction` in subclass. Add TODO